app/_how-tos/get-started-with-metering-and-billing.md

```diff
@@ -0,0 +1,253 @@
+
+<!--vale off-->
+{% entity_examples %}
+entities:
+  plugins:
+    - name: rate-limiting
+      consumer: kong-air
+      config:
+        month: 1000
+    - name: rate-limiting
+      consumer: kong-travel
+      config:
+        month: 5000
+{% endentity_examples %}
+<!--vale on-->
+
+## Configure ACL plugin
+
+Configure the ACL plugin to only allow access to the service for customers who have signed up, so our two consumers:
+
+<!--vale off-->
+{% entity_examples %}
+entities:
+  consumers:
+    - username: kong-air
+      acls:
+      - group: kong-air
+      keyauth_credentials:
+      - key: air-key
+
+    - username: kong-travel
+      acls:
+      - group: kong-travel
+      keyauth_credentials:
+      - key: travel-key
+{% endentity_examples %}
+<!--vale on-->
+
+^ ChatGPT recommended this to fix my ACL error, tested and it seems to work.
+
+## Create a meter
+
+1. In the {{site.konnect_short_name}} sidebar, click **Metering & Billing**.
+1. Enable **API Gateway Requests**.
+
+This will pull in request proxied by your API Gateway in {{site.konnect_short_name}} to Metering & Billing.
+
+## Create a feature
+
+Creating a feature that is linked to our service:
+
+1. In the {{site.konnect_short_name}} sidebar, click **Metering & Billing**.
+1. In the Metering & Billing sidebar, click **Product Catalog**.
+1. Click **Create Feature**.
+1. In the **Name** field, enter `example-service`.
+1. From the **Meter** dropdown menu, select "kong_konnect_api_request". 
+1. Click **Add group by filter**.
+1. From the **Group by** dropdown menu, select "service_name".
+1. From the **Operator** dropdown menu, select "Equals".
+1. From the **Value** dropdown menu, select "example-service".
+1. Click **Save**. 
+
+## Create a plan
+
+Creating plans for our feature.
+
+1. In the {{site.konnect_short_name}} sidebar, click **Metering & Billing**.
+1. In the Metering & Billing sidebar, click **Product Catalog**.
+1. Click the **Plans** tab.
+1. Click **Create Plan**.
+1. In the **Name** field, enter `Free`.
+1. In the **Billing cadence** dropdown menu, select "1 month".
+1. Click **Save**.
+1. Click **Add Rate Card**.
+1. From the **Feature** dropdown menu, select "example-service".
+1. Click **Next**.
+1. From the **Pricing model** dropdown menu, select "Free".
+1. Click **Next**.
+1. From the **Entitlements** dropdown, select "Metered".
+1. From the **Usage Period Interval** dropdown, select "Monthly".
+1. In the **Allowance for Period** field, enter `1000`. 
+1. Click **Save Rate Card**.
+1. Click **Publish Plan**.
+1. Navigate back to **Plans** in the breadcrumbs.
+1. Click **Create Plan**.
+1. In the **Name** field, enter `Premium`.
+1. In the **Billing cadence** dropdown menu, select "1 month".
+1. Click **Save**.
+1. Click **Add Rate Card**.
+1. From the **Feature** dropdown menu, select "example-service".
+1. Click **Next**.
+1. From the **Pricing model** dropdown menu, select "Package".
```

Comment:
IMHO selecting "Usage Based" would be simpler and more suitable for getting started.